components/customs/header.tsx

```diff
@@ -54,98 +55,112 @@ const arbotracker: navMenuItem[] = [
+        {props.navItems.map((navItem: navMenuItem) => (
+          <ListItem key={`${props.title}-${navItem.title}`} title={navItem.title} href={navItem.href}>
+            {navItem.description}
+          </ListItem>
+        ))}
+      </ul>
+    </div>
+  )
+}
+
 export const Header = () => {
-  const [language, setLanguage] = useState<"en" | "fr" | "de">("en");
+
+  const pathname = usePathname();
+  const [titleSuffix, setTitleSuffix] = useState("Sero");
+  const [titleSuffixColor, setTitleSuffixColor] = useState("text-background");
+  const [headerBgColor, setHeaderBgColor] = useState("bg-background")
+
+
+  useEffect(() => {
+    if (pathname.includes('arbovirus')) {
+      setTitleSuffix("Arbo");
+      setTitleSuffixColor("text-arbovirus");
+      setHeaderBgColor("bg-arbovirus delay-150");
+    } else if (pathname.includes('sarscov2')) {
+      setTitleSuffix("SC2");
+      setTitleSuffixColor("text-sc2virus");
+      setHeaderBgColor("bg-sc2virus delay-150");
+    } else {
+      setTitleSuffix("Sero");
+      setTitleSuffixColor("text-background");
+      setHeaderBgColor("bg-background");
+    }
+  }, [pathname])
 
   return (
-    <header className="bg-background flex items-center justify-between h-12 w-screen px-2">
-      <div className="cursor-pointer py-5 pl-2">
-        <Link href={"/"} className="flex items-center">
-          <Image
-            src={"/SerotrackerLogo.svg"}
-            alt={"Serotracker Logo"}
-            width={23}
-            height={23}
-          />
+    <header className={cn("flex items-center  justify-between transition-colors duration-300 h-14 w-screen px-2 text-white border-b-4 border-white overflow-hidden", headerBgColor)}>
+      <div className="cursor-pointer pl-2">
+        <Link href={"/"} className="flex items-center text-h1">
+          <h2>
+            <span className={cn("p-1 mr-1 bg-white rounded-md transition-colors duration-300", titleSuffixColor)}>{titleSuffix}</span>
+            Tracker
+          </h2>
         </Link>
       </div>
-
       <NavigationMenu>
         <NavigationMenuList>
-          <NavigationMenuItem hidden={!process.env.SARS_COV_2_TRACKER_ENABLED}>
-            <NavigationMenuTrigger>SeroTracker</NavigationMenuTrigger>
+          <NavigationMenuItem>
+            <NavigationMenuTrigger className="bg-transparent">Trackers</NavigationMenuTrigger>
             <NavigationMenuContent>
-              <ul className="grid w-[400px] gap-3 p-4 md:w-[500px] md:grid-cols-2 lg:w-[600px] ">
-                {serotracker.map((page) => (
-                  <ListItem
-                    key={page.title}
-                    title={page.title}
-                    href={page.href}
-                  >
-                    {page.description}
-                  </ListItem>
-                ))}
-              </ul>
+              <div className="p-4 flex justify-center w-full">
+                {process.env.SARS_COV_2_TRACKER_ENABLED &&
```

Comment:
I was wodnering whats going on
what do u mean?

Reply:
Yeah fair enough I explained this bad.

NextJS stores environment variables but it isn't sure which ones are public ones that the client can use and which ones are secrets that should only be used serverside.

The way it can tell the difference is whether the environment variable starts with `NEXT_PUBLIC` or not. If it starts with `NEXT_PUBLIC`, the client is allowed to access it, if it doesn't, the client isn't. I assume that my old code was getting recognized as server side code and that's why I was getting away with `SARS_COV_2_TRACKER_ENABLED` but it looks like your code is getting rendered client side so it will always be `undefined` since it's not a public env variable.

I feel like if we change this feature flag to `NEXT_PUBLIC_SARS_COV_2_TRACKER_ENABLED` everything will start working again.